### PR TITLE
Added sandia functionality to PVgen

### PR DESCRIPTION
**Pull request by antonszilasi**
@mostaphaRoudsari  ok this one should work!

### CODE REVIEW

File: src/Honeybee_Honeybee.py [hunk 1]
```diff
@@ -7986,7 +8008,7 @@ def checkGHPythonVersion(target = "0.6.0.3"):
         sc.sticky["honeybee_folders"]["DSLibPath"] = hb_DSLibPath
     
         # supported versions for EnergyPlus
-        EPVersions = ["V8-3-0", "V8-2-10", "V8-2-9", "V8-2-8", "V8-2-7", "V8-2-6", \
+        EPVersions = ["V8-4-0","V8-3-0", "V8-2-10", "V8-2-9", "V8-2-8", "V8-2-7", "V8-2-6", \
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Honeybee.py]: @antonszilasi have you tested Honeybee against "V8-4-0"?

**Comment by mostaphaRoudsari** [comment]
@antonszilasi thanks! this looks great. I think Honeybee_Generator_PV.ghuser is missing from the pull request. Also as I mentioned have you tested Honeybee against the new version of EnergyPlus?

File: src/Honeybee_Generator_PV.py [hunk 2]
```diff
@@ -57,7 +57,7 @@
 
 ghenv.Component.Name = "Honeybee_Generator_PV"
 ghenv.Component.NickName = 'PVgen'
-ghenv.Component.Message = 'VER 0.0.58\nNOV_25_2015'
+ghenv.Component.Message = 'VER 0.0.58\nJAN_16_2015'
```
Comment (mostaphaRoudsari) [inline src/Honeybee_Generator_PV.py]: Here is why the ghuser is missing. The date is wrong and Ladybug doesn't overwrite the current component which it thinks is a newer version! It should be nJAN_16_201**6**

**Comment by mostaphaRoudsari** [comment]
@antonszilasi since I need to make changes to Honeybee_Honeybee tonight and I don't want to make extra work for you while moving out, I'll merge this in and will take care of the missing userobject. Thanks again.

File: src/Honeybee_ Run Energy Simulation.py [hunk 3]
```diff
@@ -61,7 +61,7 @@
 """
 ghenv.Component.Name = "Honeybee_ Run Energy Simulation"
 ghenv.Component.NickName = 'runEnergySimulation'
-ghenv.Component.Message = 'VER 0.0.58\nJAN_02_2016'
+ghenv.Component.Message = 'VER 0.0.58\nJAN_16_2016'
 ghenv.Component.Category = "Honeybee"
 ghenv.Component.SubCategory = "09 | Energy | Energy"
 #compatibleHBVersion = VER 0.0.56\nDEC_28_2015
```
Comment (mostaphaRoudsari) [inline src/Honeybee_ Run Energy Simulation.py]: Don't forget to update compatible Honeybee version! This is a note for all of your components.

File: src/Honeybee_Honeybee.py (hunk 1 above)
Comment (antonszilasi) [inline src/Honeybee_Honeybee.py]: @mostaphaRoudsari  Yes I have

**Pull request merged**